packages/back-end/src/api/openapi/payload-schemas/PostProjectPayload.yaml

```diff
@@ -0,0 +1,8 @@
+type: object
+required:
+  - name
+properties:
+  name:
+    type: string
+  description:
+    type: string
```

Comment:
Should we allow setting the stats engine during creation?  Seems weird to require 2 API calls to do that.